Documentation/ceph-object-multisite.md

```diff
@@ -55,16 +53,17 @@ If an admin wants to sync data from another cluster, the admin needs to pull a r
 
 To begin doing this, the admin needs 2 pieces of information:
 
-	- An endpoint from the realm you are pulling from
-	- The access key and the system key of the system user from the realm you're pulling from.
+1. An endpoint from the realm being pulled from
+1. The access key and the system key of the system user from the realm being pulled from.
 
 ## Getting the Pull Endpoint
 
-To pull a Ceph realm from a remote Ceph cluster, an endpoint must be added to the CephObjectRealm's `pull` section in the `spec`. This endpoint must be from the master zone in the master zone group of that realm. 
+To pull a Ceph realm from a remote Ceph cluster, an `endpoint` must be added to the CephObjectRealm's `pull` section in the `spec`. This endpoint must be from the master zone in the master zone group of that realm. 
+
+To find such an endpoint via the tool box run:
 
-To find an endpoint in the via tool boxes run:
-``console
-radosgw-admin zonegroup get --rgw-realm=REALM_NAME --rgw-zonegroup=MASTER_ZONEGROUP_NAME
+```
+$ radosgw-admin zonegroup get --rgw-realm=$REALM_NAME --rgw-zonegroup=$MASTER_ZONEGROUP_NAME
```

Comment:
I'd not expect this command to work if the cluster is external, the key used to set up the external cluster might like privileged.

Reply:
Maybe my comment isn't clear. This command is supposed to be used **on** the cluster you're trying to pull from (it could be an external cluster) to get an endpoint. I changed the text to make it a little more clear. 

This step is not necessary if you already know the endpoint.